### DISCUSSION TITLE
Pentium 3 is simply Pentium 2 with SSE.

### DISCUSSION THREAD

**Discussion by LeoKids** [86Box/86Box#3258] [General]
I know they wouldn't implement it, but what they didn't know is that Pentium 3 is simply Pentium 2 with SSE.

**Comment by daemonspudguy** [comment]
Yes. And?

**Comment by dhrdlicka** [comment]
We know. The reason we don't plan Pentium III emulation is performance concerns.

https://86box.net/2022/03/21/why-not-p3.html
